arctic/store/_ndarray_store.py

```diff
@@ -163,6 +160,41 @@ def get_info(self, version):
         ret['rows'] = int(version['up_to'])
         return ret
 
+    def chunked_read(self, arctic_lib, version, symbol, date_range, read_preference=None):
+        collection = arctic_lib.get_top_level_collection()
+        if read_preference:
+            collection = collection.with_options(read_preference=read_preference)
+        return self._do_chunked_read(collection, version, symbol, date_range)
+
+    def _do_chunked_read(self, collection, version, symbol, date_range):
+        segment_count = version.get('segment_count', None)
+
+        spec = {'symbol': symbol,
+                'parent': version.get('base_version_id', version['_id']),
+                }
+
+        if date_range is not None:
+            spec['segment'] = date_range.mongo_query()
```

Comment:
Is this right?

If a query spans some DateRange: t -> t' 
then the query will be: `{segment: {'$gt': t, '$lt': t'}}`

If there is to segment=`t0` which chunk, which contains data for `t`, `t0` will be less than `t` and won't be picked up by the query, will it?